tools/latest-kernel-full-config.sh

```diff
@@ -213,3 +184,71 @@ validate_kernel_configs() {
         bail "Kernel configuration validation failed"
     fi
 }
+
+# Parse command line arguments
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        -h|--help)
+            usage
+            exit 0
+            ;;
+        *)
+            echo "Unknown option: $1"
+            usage
+            exit 1
+            ;;
+    esac
+    shift
+done
+
+# Ensure this script is run within the Bottlerocket SDK container
+if [[ ! -d "${KERNEL_KIT_DIR}" ]]; then
+    usage
+    bail
+fi
+
+# In kernel kit automation, CodePipeline updates each kernel separately.
+# This script processes only kernels that have available RPM files,
+# skipping any kernel packages without corresponding source RPMs.
+kernels_to_process=()
+for kernel_dir in "${KERNEL_KIT_DIR}/packages"/kernel-*; do
+    if [[ -d "${kernel_dir}" ]]; then
+        kernel_pkg=$(basename "${kernel_dir}")
+        rpm_file=$(find "${kernel_dir}" -name "kernel*.src.rpm" | head -1)
```

Comment:
This should get the latest RPM if you use `tail`:


```suggestion
        rpm_file=$(find "${kernel_dir}" -name "kernel*.src.rpm" | tail -1)
```